apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/BigtableClientPartial.cs

```diff
@@ -1000,116 +973,39 @@ private static MutateRowsRequest CreateMutateRowRequest(TableName tableName, IEn
-                BigtableServiceApiSettings bigtableServiceApiSettings = null,
-                ChannelCredentials credentials = null,
-                ServiceEndpoint serviceEndpoint = null)
-            {
-                ClientCount = clientCount;
-                BigtableServiceApiSettings = bigtableServiceApiSettings;
-                Credentials = credentials;
-                ServiceEndpoint = serviceEndpoint;
-            }
-
-            /// <summary>
-            /// The number of <see cref="BigtableServiceApiClient"/>s to create and use within a <see cref="BigtableClient"/> instance.
-            /// If <c>null</c>, defaults to the CPU count on the machine this is being executed on.
-            /// </summary>
-            public int? ClientCount { get; }
-
-            /// <summary>
-            /// The settings to use when creating <see cref="BigtableServiceApiClient"/> instances.
-            /// If <c>null</c>, defaults to <see2 cref="BigtableServiceApiSettings.GetDefault"/>.
-            /// </summary>
-            public BigtableServiceApiSettings BigtableServiceApiSettings { get; }
-
-            /// <summary>
-            /// Credentials to use when creating <see cref="BigtableServiceApiClient"/> instances.
-            /// If <c>null</c>, defaults to using the default credentials.
-            /// </summary>
-            public ChannelCredentials Credentials { get; }
-
-            /// <summary>
-            /// The <see cref="ServiceEndpoint"/> to use when creating <see cref="BigtableServiceApiClient"/> instances.
-            /// If <c>null</c>, defaults to <see cref="BigtableServiceApiClient.DefaultEndpoint"/>.
-            /// </summary>
-            public ServiceEndpoint ServiceEndpoint { get; }
-
-            /// <summary>
-            /// Creates <see cref="ClientCreationSettings"/> to be used in creation of <see cref="BigtableClient"/> for use with Emulator.
-            /// <param name="target"> Service enpoint target in the yourhost:yourport form.</param>
-            /// </summary>
-            internal static ClientCreationSettings FromEndpointTarget(string target)
-            {
-                string[] endpointValues = target.Split(':');
-                return new ClientCreationSettings(null, null, ChannelCredentials.Insecure, new ServiceEndpoint(endpointValues[0], Convert.ToInt32(endpointValues[1])));
-            }
-
-            internal void Validate()
-            {
-                // Fairly arbitrary upper limit.
-                GaxPreconditions.CheckArgumentRange(ClientCount ?? 1, nameof(ClientCount), 1, 256);
-            }
-        }
     }
 
     public partial class BigtableClientImpl
     {
         private readonly string _appProfileId;
-        private readonly BigtableServiceApiClient[] _clients;
-        private int _clientNumber = -1;
+        private readonly BigtableServiceApiClient _client;
 
         // TODO: Add a public constructor after multi-channel support?
```

Comment:
Remove TODO?

Reply:
I was going to review all existing "multi-channel" TODOs after this support was already in place since this PR was big enough as-is.